# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: browser/components/customizableui/test/browser_962069_drag_to_overflow_chevron.js
# Commit: beeda5ef3712
# Full Hash: beeda5ef3712b73a19c7a4ce08a52ab4dd08b802
# Author: David P <daparks@mozilla.com>
# Date: 2024-07-04 16:09:52
# Regressor Bug: 1893119
# File Overlap Count: 1
# Description:
#   Bug 1893119: Part 20 - Move EndDragSession from nsIDragService to nsIDragSession r=gstoll,rkraesig,win-reviewers,geckoview-reviewers,m_kato
#   
#   This is a straightforward move.  It does take the liberty of breaking out an
#   EndDragSessionImpl method, which will be needed later.
#   
# ==============================================================================

diff -r 6c3ffc8c534c -r beeda5ef3712 browser/components/customizableui/test/browser_962069_drag_to_overflow_chevron.js
--- a/browser/components/customizableui/test/browser_962069_drag_to_overflow_chevron.js	Thu Jul 04 07:48:10 2024 +0000
+++ b/browser/components/customizableui/test/browser_962069_drag_to_overflow_chevron.js	Thu Jul 04 07:48:11 2024 +0000
@@ -60,7 +60,7 @@
       overflowChevron
     );
   } finally {
-    ds.endDragSession(true);
+    ds.getCurrentSession().endDragSession(true);
   }
 
   info("Overflow panel is shown.");